community/en-us/join/e2e-guide.md

```diff
@@ -29,7 +29,7 @@ In each issue, we list the content to be tested and the expected results, which
 
 ### How to write test code?
 
-After taking up the corresponding task, the next step is to get to the crux of writing the code. Many partners may not be familiar with E2E test codes, so you can refer to this page: [e2e-test](../../../development/zh-cn/e2e-test.md).
+After taking up the corresponding task, the next step is to get to the crux of writing the code. Many partners may not be familiar with E2E test codes, so you can refer to this page: [e2e-test](../../../en-us/development/e2e-test.md).
```

Comment:
I take a quick look, our document will change the path when we build from source. maybe we should add full http when we need to link between `community` and `developer` tab, just like `[Contributing Guides](https://dolphinscheduler.apache.org/en-us/community/development/contribute.html)` in https://dolphinscheduler.apache.org/en-us/community/team.html

Reply:
I mean, we should use the link `[e2e-test](https://dolphinscheduler.apache.org/en-us/development/e2e-test.html)`

Reply:
> I mean, we should use the link `[e2e-test](https://dolphinscheduler.apache.org/en-us/development/e2e-test.html)`

I have updated it.